releasenotes/notes/refactor-basicaer-to-basicprovider-e27aff9c8e81d26e.yaml

```diff
@@ -0,0 +1,90 @@
+---
+upgrade:
+  - |
+    The ``qiskit.providers.basicaer`` module and all of its classes have been
+    removed following their deprecation on the 0.46 release. They have been
+    replaced by the :mod:`qiskit.quantum_info` module and the new
+    :mod:`qiskit.providers.basic_provider` module.
```

Comment:
Did you want to mention `qiskit.BasicAer` here being removed too?

Reply:
Done!